doc/ref/fpsemi.xml

```diff
@@ -34,12 +35,12 @@ Finitely presented monoids are obtained by factoring a free monoid by
 a set of relations, i.e. a set of pairs of words in the free monoid. 
 <P/>
 <Example><![CDATA[
-gap> f:=FreeMonoid("a","b");;
-gap> x:=GeneratorsOfMonoid(f);
+gap> f := FreeMonoid( "a", "b" );;
+gap> x := GeneratorsOfMonoid(f);
```

Comment:
I'm unfamiliar with the spacing conventions in the reference manual, assuming such exist and are rigorously adhered to, but will investigate.